priv/scim.abnf

```diff
@@ -34,7 +36,7 @@ subAttr   = "." ATTRNAME
 	  ; a sub-attribute of a complex attribute
 
 ;;
-scim-rfc-path = attrPath / valuePath [subAttr]
+scim-rfc-path = valuePath [subAttr] / attrPath
```

Comment:
Because of aforementioned lack of backtracking, we need the longest matching sequence go first. Otherwise, `attrPath` always matches the prefix of `valuePath` and the `valuePath` branch is never reached.